### ISSUE TITLE
Implementing other (images &videos) divs in "All" ciontainer 

### ISSUE THREAD

**Issue by Aakash-DD** [fossasia/susper.com#1420]
**I'm submitting a ...**
- [ ] feature request

**Current behavior:**

 No divs created for other features as well
**Expected behavior:**

Creating Some divs(parts) for -
1. Images
2. Videos
in the "All" container

**Screenshots of the issue:**

**For images -**

![screenshot 47 _li](https://user-images.githubusercontent.com/33255535/51556304-d72a0780-1e9f-11e9-9e96-8a5ff05ea03d.jpg)

**For videos -**

![screenshot 46 _li](https://user-images.githubusercontent.com/33255535/51556338-e7da7d80-1e9f-11e9-9d29-70f55f292de5.jpg)
**Will work on the issue:**
yes

**Comment by Aakash-DD** [comment]
This will give a overall more attractive look to the design of Susper ,
where user will also find useful to navigate easily.

**Comment by subhahu123** [comment]
@Aakash-DD can i work on this

**Comment by Aakash-DD** [comment]
@subhahu123 me working on this...
waiting just only for mentors to confirm.... @praveenojha33 @simsausaurabh
(And then see the magic 😅 )

**Comment by praveenojha33** [comment]
@Aakash-DD Yeah It will be great if you can implement this feature using YaCy only.

**Comment by Aakash-DD** [comment]
sorry man, i m busy in some hackathon right now.....some one @subhahu123  u might do this or @mentors i will fix this after some days.
